### PR TITLE
[WIP] Docker implementation

### PR DESCRIPTION
**Pull request by eiguike**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM postgres
+COPY backup.backup /docker-entrypoint-initdb.d/
+FROM ruby:2.2.6
```
Comment (Jostavo) [inline Dockerfile]: Wait is that right?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_USER=docker
+      - POSTGRES_PASSWORD=docker
+      - POSTGRES_DB=siglateste
```
Comment (Jostavo) [inline docker-compose.yml]: I changed db names on my PR, should I rollback to the other name or we fix it later?

File: config/database.yml [hunk 3]
```diff
@@ -12,7 +12,10 @@ default: &default
 
 development:
   <<: *default
-  database: siglateste
+  # database: siglateste
+  host: db
+  username: postgres
+  password:
```
Comment (Jostavo) [inline config/database.yml]: I'm using postgres as password to the database

File: Dockerfile (hunk 1 above)
Comment (eiguike) [reply Dockerfile]: Actually, I'm not sure which version is correct to be set.

File: docker-compose.yml (hunk 2 above)
Comment (eiguike) [reply docker-compose.yml]: No, continue with your changes, I'll update this with yours.

File: config/database.yml (hunk 3 above)
Comment (eiguike) [reply config/database.yml]: Hmm, I'll change it! Thanks for the tip!

**Comment by eiguike** [comment]
@Darkkgreen!!11

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6.3
```
Comment (Jostavo) [inline Dockerfile:1]: Is this the right version? Not sure if we should use 2.4.1 or this one. If we're using this one I'll have to update my CI/CD config files

**Review by Jostavo — APPROVED** [review]

File: Dockerfile (hunk 4 above)
Comment (eiguike) [reply Dockerfile:1]: I didn't have any problems to use the latest version, I think it is safe to do this move.

**Pull request merged**